src/rgw/store/sfs/CMakeLists.txt

```diff
@@ -0,0 +1,53 @@
+#need to update cmake version here
+cmake_minimum_required(VERSION 3.14.0)
```

Comment:
is this version different from the rest of the ceph project?

Reply:
no

Reply:
ups. sorry I had a filter in the search, yes it is different in many places.

Reply:
are you concerned for something in particular?

Reply:
no, just wondered because of the comment above the `required` line. :) as long as it doesn't introduce a new cmake version to the remaining project, I'm okay with it.